src/core/thread/mesh_forwarder.cpp

```diff
@@ -2271,4 +2279,75 @@ void MeshForwarder::HandleDataPollTimeout(void)
     }
 }
 
+#if OPENTHREAD_CONFIG_LOG_LEVEL >= OPENTHREAD_LOG_LEVEL_INFO
```

Comment:
I agree. We may need to later consider adding a new log level (like `notice` in between `info` and `warning`)...or another option is to add a `CONFIG' option to enable/disable logs per IP message.